hibike/lib/hibike/hibike_device.h

```diff
@@ -35,12 +36,8 @@ void hibike_setup();
 void hibike_loop();
 void toggleLED();
 
-extern uint32_t device_update(uint8_t param, uint32_t value); 
-
-extern uint32_t device_status(uint8_t param); 
-
-extern uint8_t data_update(uint8_t* data_update_buf, size_t buf_len);
-
+extern uint32_t device_write(uint8_t param, uint8_t* data,size_t len);
+extern uint8_t device_data_update(uint8_t param, uint8_t* data_update_buf, size_t buf_len);
```

Comment:
I think this should be renamed to `device_read()` and have the same argument names as `device_write()` to make things cleaner. This change would be need to be made in multiple places:
 - The hibike firmware library
 - Firmware for each specific device
 - DEVELOPERS.md